tldw_Server_API/app/core/AuthNZ/repos/usage_repo.py

```
@@ -0,0 +1,290 @@
+                    start_str,
+                    end_str,
+                    key_id,
+                )
+
+            tokens = 0
+            usd = 0.0
+            if row:
+                tokens = int(row.get("tokens") or 0)
+                usd = float(row.get("usd") or 0.0)
+
+            return {"tokens": tokens, "usd": usd}
```

Ruff (0.14.6)

Consider moving this statement to an `else` block

(TRY300)